articles/openstack/ostack-how-static-ip.md

```diff
@@ -0,0 +1,175 @@
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create Static IPs for assignment to instances.
+toc_fullpath: How To/ostack-how-static-ip.md
+toc_mdlink: ostack-how-static-ip.md
+---
+
+# Creating Static IPs for assignment to instances.
+
+Static IPs  are assigned to instances in openstack by creating ports and adding them to the instance.
+
+A port is a connection point for attaching a single device, such as the NIC of a server, to a network. The port also describes the associated network configuration, such as the MAC and IP addresses to be used on that port.
+
+## Steps
+
+> ![NOTE]
+> You will need to source your RC file before you run the commands below.
+
+### Create a network
+```
+$ openstack network create testnet
+
++---------------------------+--------------------------------------+
+| Field                     | Value                                |
++---------------------------+--------------------------------------+
+| admin_state_up            | UP                                   |
+| availability_zone_hints   |                                      |
+| availability_zones        |                                      |
+| created_at                | 2018-09-05T12:20:30Z                 |
+| description               |                                      |
+| dns_domain                | None                                 |
+| id                        | dd4f624a-1101-4b01-a388-308c5972742a |
+| ipv4_address_scope        | None                                 |
+| ipv6_address_scope        | None                                 |
+| is_default                | None                                 |
+| is_vlan_transparent       | None                                 |
+| mtu                       | 1446                                 |
+| name                      | testnet                              |
+| port_security_enabled     | True                                 |
+| project_id                | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx     |
+| provider:network_type     | None                                 |
+| provider:physical_network | None                                 |
+| provider:segmentation_id  | None                                 |
+| qos_policy_id             | None                                 |
+| revision_number           | 3                                    |
+| router:external           | Internal                             |
+| segments                  | None                                 |
+| shared                    | False                                |
+| status                    | ACTIVE                               |
+| subnets                   |                                      |
+| tags                      |                                      |
+| updated_at                | 2018-09-05T12:20:30Z                 |
++---------------------------+--------------------------------------+
+```
+### Create a subnet and attach to the network
```

Comment:
2, Create a subnet and attach it to the network by running the following command:

$ openstack subnet create --network testnet --subnet-range 10.1.1.0/24 testnet-sub

This returns the following: